homeassistant/components/tts/media_source.py

```diff
@@ -104,7 +104,7 @@ def media_source_id_to_kwargs(media_source_id: str) -> MediaSourceOptions:
 class TTSMediaSource(MediaSource):
     """Provide text-to-speech providers as media sources."""
 
-    name: str = "Text to Speech"
+    name: str = "Text-to-speech"
```

Comment:
@MartinHjelmare Would anything be relying on this name?

Reply:
Maybe, I didn't look into the details yet, but since we have a test for it I suggest we look at it and change it separately.

Reply:
Oh, we did change this I see now.

Reply:
This will change the title of the `BrowseMedia` item we send to the frontend.

Reply:
@balloob are you confident about that change?